### PR TITLE
Adds documentation for helper functions

### PR DESCRIPTION
**Pull request by harshavamsi**
Signed-off-by: Harsha Vamsi Kalluri <harshavamsi096@gmail.com>

### Description
Adds documentation for various methods in the helper library

### Issues Resolved
Closes #187 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: docs/source/api-ref/helpers.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# helpers
```
[the remark this answers is not in the document]
Comment (harshavamsi) [reply docs/source/api-ref/helpers.md:1]: I've separated the two methods.

File: dev-requirements.txt [hunk 2]
```diff
@@ -3,9 +3,10 @@ pytest
 pytest-cov
 coverage
 mock
-sphinx<1.7
+sphinx>=2.0.0
 sphinx_rtd_theme
-jinja2
+jinja2<3
+markupsafe==2.0.1
```
Comment (axeoman) [inline dev-requirements.txt:9]: CI is not happy with these changes (test-2.7 and test-3.5) and according to the PR title and commits message it shouldn't be here.

**Review by dblock — APPROVED** [review]

**Comment by dblock** [comment]
@harshavamsi Want to finish this?

**Comment by wbeckler** [comment]
I'm going to close this for now, and if needed we can try again without the requirements changes.

**Pull request closed without merging**